src/azurestorage.cpp

```diff
@@ -18,11 +18,28 @@
 #include <memory>
 
 #include "azurefilesystem.hpp"
+#include "logging.hpp"
 
 namespace ovms {
 
 using namespace utility;
 
+void print_as_storage_exception_msg(const as::storage_exception& e){
+    as::request_result result = e.result();
+    as::storage_extended_error extended_error = result.extended_error();
+    if (!extended_error.message().empty()) {
+        SPDLOG_LOGGER_ERROR(azurestorage_logger, "Unable to access path: {}", extended_error.message());
```

Comment:
I would prefer to have SPDLOG_LOGGER_ERROR left as it is and have sth like:
SPDLOG_LOGGER_ERROR(azurestorage_logger, "Unable to access path: {}", extractErrorMessage(e));
this way you keep line in log, and encapsulate logic that you want.